connector/connect/client/go/README.md

```diff
@@ -0,0 +1,35 @@
+- Prepare your environment to generate proto Go files:
```

Comment:
The current buf build should support generating the go bindings already. I think we can find an easy way to make it work.

Reply:
Do you mean the `protobuf-maven-plugin` plugin? Unfortunately it does not support generating go bindings. I also searched around, did not find proper plugins in maven to generate protobuf go code.

Reply:
We have been using this to generate proto code for python https://github.com/apache/spark/blob/master/dev/connect-gen-protos.sh

Reply:
Cool, thanks for the info!